packages/metastream-app/src/assets/homescreen.html

```diff
@@ -86,7 +86,7 @@
     <header id="tips" class="tips">
       <button id="tipsbtn" class="tips-button">
         <div class="column">
-          <b>📝 Tips for getting started</b>
+          <b>📝 {t('tipsGettingStarted1')}</b>
```

Comment:
This particular file will be tricky. It didn't have translations because it's using plain HTML. Before translations can be easily supported, it will need to be converted to a React component with JSX markup.

This is possible, but I might not be able to get around to refactoring this during the week.